nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -669,6 +669,22 @@ def _impl(inputs, in_state_c, in_state_h, attr, params):
     return _impl
 
 
+def _pad(name):
+    def _impl(inputs, attr, params):
+        padlist = params.pop(inputs[1].list_output_names()[0]).asnumpy()
```

Comment:
Suggest to assert if padlist is not available in params.

Reply:
Please assert for mode not equal to ```CONSTANT```
And the mode string is case-insensitive.

Reply:
@srkreddy1238 Sorry for the delay. In terms of wether I have use cases to use `MirrorPad` (use tf.pad with symmetric or reflect mode), I don't have any data at the moment.  But, since `Pad` is required for resnet, I want to add `Pad` support for now.

To support `MirrorPad`, we will have to add `mode` option(attribute?) to pad operator (in nnvm and perhaps tvm) to distinguish behavior of padding(`CONSTANT`, `REFLECT`, `SYMMETRIC`) . Since `MirrorPad` has `mode` as attribute to distinguish between `REFLECT` and `SYMMETRIC`, we will be able to pass it to the pad operator.

I might have a misunderstand so please let me if we can give you any other information.

Reply:
@nishi-t don't worry about  MirrorPad, I just asked about it to know the requirement of it.